Makefile

```diff
@@ -44,7 +44,7 @@ endif
 IMAGE_PLATFORMS ?= linux/amd64,linux/arm64
 
 # Base build image to use.
-BUILD_BASE_IMAGE ?= golang:1.22.5@sha256:829eff99a4b2abffe68f6a3847337bf6455d69d17e49ec1a97dac78834754bd6
+BUILD_BASE_IMAGE ?= golang:1.22.8@sha256:9f7fab27a7199abd0902daf34abf1e93aca4bafda4228bd303fdd7b4cbfab082
```

Comment:
nit: udpate to same tag as https://github.com/projectcontour/contour/pull/6715 just for consistency

Reply:
Done!